### PR TITLE
[v18.2.x] v18.2.1 backports, release notes

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 Release is now available from:
 
   <https://www.dash.org/downloads/#wallets>
 
-This is a new minor version release, bringing new features, various bugfixes
-and other improvements.
+This is a new hotfix version release, bringing various bugfixes.
 
-This release is optional for all nodes.
+Please note that v18.2.0 was revoked due to a bug, this version fixes that bug.
```
Comment (thephez) [inline doc/release-notes.md:10]:
```suggestion
Please note that v18.2.0 was revoked due to a bug; this version fixes that bug.
```

File: doc/release-notes.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+
+This release is optional for all nodes. However, v18.2.1 is required to be 
+able to use both mainnet or testnet. Currently, v18.2.0 is not working on mainnet,
```
Comment (thephez) [inline doc/release-notes.md:13]:
```suggestion
able to use both mainnet and testnet. Currently, v18.2.0 is not working on mainnet,
```

File: doc/release-notes.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+and v18.1.1 is not working on testnet; v18.2.1 will work on both networks.
 
 Please report bugs using the issue tracker at github:
```
Comment (thephez) [inline doc/release-notes.md:16]:
```suggestion
Please report bugs using the issue tracker at GitHub:
```

File: doc/release-notes.md [hunk 4]
```diff
@@ -38,16 +41,21 @@ downgrade to an older version is only possible with a reindex
...
 
 ### Downgrade to a version < v18.0.1
 
 Downgrading to a version older than v18.0.1 is not supported due to changes in
 the indexes database folder. If you need to use an older version, you must
 either reindex or re-sync the whole chain.
 
+### Downgrade of masternodes to < 18.2.1
+
+It is highly recommended to not downgrade masternodes below 18.2.1, as 18.2.1 (and 18.1.1)
```
Comment (thephez) [inline doc/release-notes.md:56]:
```suggestion
It is highly recommended not to downgrade masternodes below 18.2.1, as 18.2.1 (and 18.1.1)
```

File: doc/release-notes.md [hunk 5]
```diff
@@ -1,14 +1,17 @@
...
   <https://www.dash.org/downloads/#wallets>
 
-This is a new minor version release, bringing new features, various bugfixes
-and other improvements.
+This is a new hotfix version release, bringing various bugfixes.
 
-This release is optional for all nodes.
+Please note that v18.2.0 was revoked due to a bug, this version fixes that bug.
+
+This release is optional for all nodes. However, v18.2.1 is required to be
```
Comment (thephez) [inline doc/release-notes.md:12]:
```suggestion
This release is optional for all nodes; however, v18.2.1 is required to be 
```

File: doc/release-notes.md [hunk 6]
```diff
@@ -64,48 +72,26 @@ releases are minimized, however we do not guarantee there are no breaking change
 Bitcoin backports often introduce breaking changes, and are a likely source of
 breaking changes in minor releases. Patch releases should never contain breaking changes.
 
-This release **does** include breaking changes. Please read below to see if they will affect you.
-
 Notable changes
 ===============
+See #5145 and #5142, these 2 PR fix important bugs in previous versions. Specifically,
```
Comment (thephez) [inline doc/release-notes.md:77]:
```suggestion
See #5145 and #5142; these 2 PR fix important bugs in previous versions. Specifically,
```

File: doc/release-notes.md [hunk 7]
```diff
@@ -64,48 +72,26 @@ releases are minimized, however we do not guarantee there are no breaking change
...
+#5145 fixes an issue where qfcommit messages can be replayed from the past, then are
+validated and propagated to other nodes. This patch changes it so that old qfcommits
```
Comment (thephez) [inline doc/release-notes.md:79]:
```suggestion
validated and propagated to other nodes. This patch prevents old qfcommits
```

File: doc/release-notes.md [hunk 8]
```diff
@@ -64,48 +72,26 @@ releases are minimized, however we do not guarantee there are no breaking change
...
+are not relayed. #5142 is a fix which enables this version to function both on testnet
```
Comment (thephez) [inline doc/release-notes.md:80]:
```suggestion
from being relayed. #5142 is a fix which enables this version to function both on testnet
```

**Review by thephez — CHANGES_REQUESTED** [review]
A few minor changes.

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**